src/libraries/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.Win32Unix.cs

```diff
@@ -59,7 +59,7 @@ internal static void GetDefaultIdentityAndHash(out object identity, out string h
                 hash = IdentityHelper.GetNormalizedStrongNameHash(assemblyName)!;
                 if (hash != null)
                 {
-                    hash = "StrongName" + separator + hash;
+                    hash = string.Concat("StrongName", new ReadOnlySpan<char>(in separator), hash);
```

Comment:
Is this something Roslyn could do?

Reply:
Will need to think about this one. But for the const, come to think of it I thought I fixed that case in Roslyn years ago... will look into it...

Reply:
Ah, I had it in my head that Path.DirectorySeparatorChar was a public const, but it's actually a public static readonly. The change I made to Roslyn years ago was about promoting const chars to const strings, and that's still in effect.

Reply:
I opened https://github.com/dotnet/roslyn/issues/66827.